tox.ini

```diff
@@ -4,12 +4,22 @@
 # and then run "tox" from this directory.
 
 [tox]
-envlist = py35, py36
+envlist = py35, py36, py37, py38, py39, py310
+
+[gh-actions]
+python = 
+    3.6: py36
+    3.7: py37
+    3.8: py38
+    3.9: py39
+    3.10: py310
+
 
 [testenv]
 deps =
     coverage
     ansicolors
+    colors.py
```

Comment:
Why do we need this package?

Reply:
You use it in your tests. (Import colors). At least the tests wouldn’t pass without it

Reply:
I see. Thank you for adding it!

Reply:
Actually, `ansicolors` is what `import colors` is referring to.